package.json

```diff
@@ -27,7 +27,7 @@
     "eslint-plugin-jsdoc": "^48.5.2",
     "eslint-plugin-prettier": "^5.1.3",
     "eslint-plugin-require-extensions": "^0.1.3",
-    "lerna": "^5.6.2",
+    "lerna": "^7.4.2",
```

Comment:
Out of curiosity, why stop at 7 and not go all the way to 8 ?

Reply:
Not wanting to take additional risk. v7 has enough to drop the patches.

https://github.com/lerna/lerna/blob/main/CHANGELOG.md#breaking-changes seems like it wouldn't require more work but it also isn't compelling